File: solutions/cpp/pangram/2/pangram.cpp
#include "pangram.h"
#include <algorithm>

namespace pangram {
    bool is_pangram(std::string sentence) {
        std::transform(sentence.begin(), sentence.end(), sentence.begin(), ::tolower);
        
        std::string alphabet = "abcdefghijklmnopqrstuvwxyz";
        
        for (char i : alphabet) {
            if (sentence.find(i) != std::string::npos) {
                continue;
            } else {
                return false;
            }
        }
        return true;
    }
}  // namespace pangram

File: solutions/python/pangram/2/pangram.py
'''Determine if sentence is pangram (includes all letters of alphabet)'''

def is_pangram(sentence):
    letters = []

    for character in sentence:
        if character.lower() not in letters and character.isalpha() is True:
            letters.append(character)

    return bool(len(letters) == 26)
        
